src/Adapter/MSTest.TestAdapter/Helpers/ReflectHelper.cs

```diff
@@ -301,20 +244,13 @@ internal virtual bool IsMethodDeclaredInSameAssemblyAsType(MethodInfo method, Ty
     /// <param name="categoryAttributeProvider">The member to inspect.</param>
     /// <param name="owningType">The reflected type that owns <paramref name="categoryAttributeProvider"/>.</param>
     /// <returns>Categories defined.</returns>
-    internal virtual string[] GetCategories(MemberInfo categoryAttributeProvider, Type owningType)
+    internal virtual string[] GetTestCategories(MemberInfo categoryAttributeProvider, Type owningType)
```

Comment:
Is is overriden somewhere?

Reply:
it is not. It was before.

Reply:
Correction, it is used by Moq and fails without it. reverted and commented.